src/main/resources/de/qabel/desktop/ui/layout.fxml

```diff
@@ -26,29 +26,17 @@
                              </Pane>
                             <VBox BorderPane.alignment="CENTER" styleClass="aliasContainer">
                                <children>
-                                  <Label fx:id="alias" text="Guy Pearce Longerman">
-                                     <font>
-                                        <Font name="System Bold" size="13.0" />
-                                     </font>
-                                  </Label>
-                                  <Label fx:id="mail" text="guy@pearce.com">
-                                      <font>
-                                          <Font name="System" size="9.5" />
```

Comment:
same as the avatar, it needs to be smaller. maybe a css class `.small` could make those styles consistent